tests/integration/services.test.js

```diff
@@ -38,40 +42,67 @@ describe.skip('UserMergeService', function () {
 
       await UserMergeService.mergeAccounts(ogUser1, ogUser2)
 
-      // TODO: get from db
-      const newParent = ogUser2
-      const [newChild1, newChild2, newChild3] = [ogUser1, mergedUser1, mergedUser2]
+      const newParent = getUser(ogUser2)
+      const newChild1 = getUser(ogUser1); const newChild2 = getUser(mergedUser1); const newChild3 = getUser(mergedUser2)
```

Comment:
Please don't put multiple assignment statements on one line.
* https://feross.org/never-use-semicolons/#:~:text=This%20post%20isn't%20intended,(ASI)%20is%20completely%20incorrect.
* http://benalman.com/news/2012/05/multiple-var-statements-javascript/